src/docbkx/chapter77-express-data-prep.xml

```diff
@@ -0,0 +1,370 @@
+                    </row>
+                    <row>
+                        <entry>psp.suppression.password</entry>
+                        <entry>password</entry>
+                        <entry>yes</entry>
+                        <entry>credentials for additional suppression services</entry>
+                    </row>
+                    <row>
+                        <entry>psp.suppression.email</entry>
+                        <entry>username@domain.com</entry>
+                        <entry>yes</entry>
+                        <entry>target e-mail for suppression services notifications</entry>
+                    </row>
+                    <row>
+                        <entry>psp.outgoing.file.path</entry>
+                        <entry>C:\outgoing</entry>
+                        <entry>no</entry>
+                        <entry>directory for output data files (valid, invalid, samples)</entry>
+                    </row>
+                    <row>
+                        <entry>psp.postalpresorting.defaulttemplatename</entry>
+                        <entry>mailmark</entry>
+                        <entry>no</entry>
+                        <entry>template for presorting service</entry>
+                    </row>
+                    <row>
+                        <entry>psp.postalpresorting.defaultfilesfolder</entry>
+                        <entry>C:\presorting</entry>
+                        <entry>no</entry>
+                        <entry>directory for presorting service output (PDF report)</entry>
+                    </row>
+                </tbody>
+            </tgroup>
+        </table>
+
+        <emphasis>DataCleaner Monitor setup</emphasis>
+        <para>
+            In order to use <emphasis>hot folder functionality</emphasis> you have to configure DataCleaner Monitor
+            connection first. After you deploy the war file and have it running in your browser
+            (see <link linkend="installing_datacleaner_monitor">DataCleaner Monitor</link> for details),
+            go to <emphasis>More/DataCleaner monitor</emphasis> menu item.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-monitor-setup-menu.png" format="PNG" scalefit="1"/>
+                </imageobject>
+            </mediaobject>
+
+            Then fill in the following form, test the connection and save it.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-monitor-setup-dialog.png" format="PNG" scalefit="1"/>
+                </imageobject>
+            </mediaobject>
+        </para>
+    </section>
+
+    <section id="edp_wizard_datastores">
+        <title>Datastores</title>
+        <emphasis>Supported formats</emphasis>
+        <para>
+            Express Data Prep Wizard supports CSV (comma separated value) and Excel data files. You need to provide
```

Comment:
Excel and Csv are required if the user wants to use the hot folder functionality